bloomfilter/tests/fpr-calc.hs

```diff
@@ -0,0 +1,184 @@
+{-# LANGUAGE ParallelListComp #-}
+module Main (main) where
+
+import qualified Data.BloomFilter as B (BitsPerEntry, BloomPolicy, BloomSize,
+                     FPR, Hashable)
+import qualified Data.BloomFilter.Blocked as B.Blocked
+import qualified Data.BloomFilter.Classic as B.Classic
+
+import           Control.Parallel.Strategies
+import           Data.IntSet (IntSet)
+import qualified Data.IntSet as IntSet
+import           Data.List (unfoldr)
+import           Math.Regression.Simple
+import           System.IO
+import           System.Random
+
+import           Prelude hiding (elem)
+
+-- | Write out data files used by gnuplot fpr.plot
+main :: IO ()
+main = do
+    hSetBuffering stdout NoBuffering --for progress reporting
+
+    withFile "bloomfilter/fpr.classic.gnuplot.data" WriteMode $ \h -> do
+      hSetBuffering h LineBuffering --for incremental output
+      mapM_ (\l -> hPutStrLn h l >> putChar '.') $
+        [ unwords [show bitsperkey, show y1, show y2]
+        | (bitsperkey, _) <- xs_classic
+        | y1              <- ys_classic_calc
+        | y2              <- ys_classic_actual
+        ]
+    putStrLn "Wrote bloomfilter/fpr.classic.gnuplot.data"
+
+    withFile "bloomfilter/fpr.blocked.gnuplot.data" WriteMode $ \h -> do
+      hSetBuffering h LineBuffering --for incremental output
+      mapM_ (\l -> hPutStrLn h l >> putChar '.') $
+        [ unwords [show bitsperkey, show y1, show y2]
+        | (bitsperkey, _) <- xs_blocked
+        | y1              <- ys_blocked_calc
+        | y2              <- ys_blocked_actual
+        ]
+    putStrLn "Wrote bloomfilter/fpr.blocked.gnuplot.data"
+
+    let regressionData     :: [(Double, Double)]
+        regressionData      = zip (map fst xs_blocked)
+                                  (map (negate . log) ys_blocked_actual)
+        regressionBitsToFPR = quadraticFit (\(x,y)->(x,y)) regressionData
+        regressionFPRToBits = quadraticFit (\(x,y)->(y,x)) regressionData
+    putStrLn ""
+    putStrLn "Blocked bloom filter quadratic regressions:"
+    putStrLn "bits indepedent, FPR depedent:"
+    print regressionBitsToFPR
+    putStrLn ""
+    putStrLn "FPR indepedent, bits depedent:"
+    print regressionFPRToBits
```

Comment:
"indepedent" -> 'independent" 
"depedent" -> "dependent"

Reply:
oops, thanks, fixed them all in the module.